stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -1059,7 +1061,8 @@ impl StacksChainState {
                               "function_name" => %contract_call.function_name,
                               "function_args" => %VecDisplay(&contract_call.function_args),
                               "return_value" => %return_value,
-                              "cost" => ?total_cost);
+                              "cost" => ?total_cost,
+                              "txid" => %tx.txid());
```

Comment:
`txid` is already in this log, see line 1057. Similarly for the other logs in this file.

Reply:
Thanks, removed this one again.